README.md

```diff
@@ -1 +1,230 @@
-# ballerina-aks-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment)
+
+# Deployment Ballerina with Azure Kubernetes Service
```

Comment:
We have to modify this title to match the other BBGs [1]. Maybe something like "Deploying on Azure Kubernetes".

[1] - https://ballerina.io/learn/by-guide/

Reply:
Yes, as discussed offline, let's go with "Ballerina Deployment with Azure Kubernetes Service".